src/SignalR/common/Protocols.Json/src/JsonHubProtocolOptions.cs

```diff
@@ -21,5 +21,20 @@ public JsonHubProtocolOptions()
         public bool IgnoreNullValues { get => _serializerOptions.IgnoreNullValues; set => _serializerOptions.IgnoreNullValues = value; }
         public bool WriteIndented { get => _serializerOptions.WriteIndented; set => _serializerOptions.WriteIndented = value; }
         public bool AllowTrailingCommas { get => _serializerOptions.AllowTrailingCommas; set => _serializerOptions.AllowTrailingCommas = value; }
+        public bool UseCamelCase
+        {
+            get => _serializerOptions.PropertyNamingPolicy == JsonNamingPolicy.CamelCase;
+            set
+            {
+                if (value)
+                {
+                    _serializerOptions.PropertyNamingPolicy = JsonNamingPolicy.CamelCase;
```

Comment:
If you want to be cool `_serializerOptions.PropertyNamingPolicy = value ? JsonNamingPolicy.CamelCase : null`